### PR TITLE
add note on cpk ff with sqlite

### PR DESCRIPTION
**Pull request by dpilch**
#### Description of changes:

add note on cpk ff with sqlite

#### Related GitHub issue #, if available:

https://github.com/aws-amplify/amplify-js/issues/10872
https://github.com/aws-amplify/amplify-js/issues/10693

Some customers are missing that CPK is not supported with SQLite. Adding an additional note to the getting started guide.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [x] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_
            
### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx [hunk 1]
```diff
@@ -46,6 +46,9 @@ DataStore.configure({
 });
 ```
 
+The SQLite storage adapter does not currently support custom primary keys.
+Disable custom primary key in `amplify/cli.json` by setting `graphQLTransformer.respectPrimaryKeyAttributesOnConnectionField` to `false`.
+
```
Comment (david-mcafee) [inline src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx:51]: Thoughts on wrapping this in the Callout component?

```suggestion
<Callout>

The SQLite storage adapter does not currently support custom primary keys.
Disable custom primary key in `amplify/cli.json` by setting `graphQLTransformer.respectPrimaryKeyAttributesOnConnectionField` to `false`.

</Callout>
```

File: src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx [hunk 2]
```diff
@@ -82,5 +85,8 @@ DataStore.configure({
...
```
Comment (david-mcafee) [inline src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx:90]: Same here:

```suggestion
<Callout>

The SQLite storage adapter does not currently support custom primary keys.
Disable custom primary key in `amplify/cli.json` by setting `graphQLTransformer.respectPrimaryKeyAttributesOnConnectionField` to `false`.

</Callout>
```

**Review by david-mcafee — DISMISSED** [review]
LGTM! Added a suggestion to use the Callout component, but it's not a blocker.

**Review by david-mcafee — APPROVED** [review]
LGTM! 🎉

**Review by svidgen — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Pull request merged**